setup.py

```diff
@@ -150,6 +150,7 @@ def get_git_sha() -> str:
         "dremio": ["sqlalchemy-dremio>=1.1.5, <1.3"],
         "drill": ["sqlalchemy-drill==0.1.dev"],
         "druid": ["pydruid>=0.6.5,<0.7"],
+        "duckdb": ["duckdb-engine==0.8.0"],
```

Comment:
@betodealmeida should this be `>=0.8.0,<1.0.0`?

Reply:
@john-bodley could be, but in general I prefer to pin `0.x` versions instead of using a range, since in theory the API is not sable and `0.8.1` could introduce breaking changes.

Reply:
On that note, 0.8.1 is out, and seems to follow semver
https://github.com/duckdb/duckdb/releases/

Reply:
I could go either way on this... trying to rationalize a preference. 

If we pin the version, and there's an issue (potentially security) then we might have a fire drill on our hands.

If we use a range, and something breaks, we might have a bug on our hands.

By the "lesser evil" metric, I think this _might_ put the range idea ahead in my mind by about the width of a frog hair, as long as they stick to semver ;)

Reply:
Yeah, I know they were planning to change how databases are returned by their dialect, so maybe we keep this frozen until they hit 1.0?

Also, this is an optional dependency, people can always install a newer version if they want.